src/libraries/CustomRevert.sol

```diff
@@ -75,13 +75,13 @@ library CustomRevert {
     }
 
     /// @notice bubble up the revert message returned by a call and revert with the selector provided
-    /// @dev this function should only be used with custom errors of the type `CustomError(address contract_, bytes revertReason)`
+    /// @dev this function should only be used with custom errors of the type `CustomError(address contract, bytes revertReason)`
```

Comment:
i actually feel like `contract` is wrong? for our `Wrap__NativeTransferFailed` the `recipient` thats emitted might not be a contract. Maybe `target` is better? And that avoids the contract keyword too

Reply:
target is good, I like that

Reply:
i like target!